pkgs/development/python-modules/manuf/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, runCommand
+, python3
+}:
+
+buildPythonPackage rec {
+  pname = "manuf";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "coolbho3k";
+    repo = "manuf";
+    rev = "${version}";
+    sha256 = "sha256-3CFs3aqwE8rZPwU1QBqAGxNHT5jg7ymG12yBD56gTNI=";
+  };
+
+  # Tests appeared to be broken
+  doCheck = false;
+
+  passthru.tests = {
+    testMacAddress = runCommand "${pname}-test" {} ''
+      ${python3.pkgs.manuf}/bin/manuf BC:EE:7B:00:00:00 > $out
+      [ "$(cat $out | tr -d '\n')" = "Vendor(manuf='ASUSTekC', manuf_long='ASUSTek COMPUTER INC.', comment=None)" ]
+    '';
+  };
+
+  meta = with lib; {
+    homepage = "https://github.com/coolbho3k/manuf";
+    description = " Parser library for Wireshark's OUI database";
+    platforms = platforms.linux;
+    license = with licenses; [ lgpl3Plus asl20 ];
+    maintainers = with maintainers; [ dsuetin ];
```

```suggestion
    maintainers = with maintainers; [ dsuetin ];
    mainProgram = "manuf";
```
I think this may be useful.

---

Done